src/com/ableton/VirtualEnv.groovy

```diff
@@ -74,11 +83,14 @@ class VirtualEnv implements Serializable {
    *
    * @param script Script context.
    *               <strong>Required value, may not be null!</strong>
-   * @param python Python version or absolute path to Python executable.
+   * @param python Python version or absolute path to Python executable. On Windows, this
+   *               should be a Cygwin-style path, but <strong>without the {@code .exe}
+   *               extension</strong>, for example: {@code /c/Python27/python}
```

Comment:
Why is the docstring for this parameter being extended in 5089c97e90f3b39e5622a88893b72a165b019fd3? Also, the documentation is not accurate, `create('python2.7')` and `create('python3.7')` work fine on Windows.

Reply:
I'll make another cleanup commit for the documentation.

Reply:
I had meant to put that in another cleanup commit, but forgot. I guess it's not adding much, though, so I'll remove it.